src/migrations/00033-add-cpu-arch-to-device-type.sql

```diff
@@ -0,0 +1,9 @@
+-- In case the cpu architecture is empty, we just add a dummy value to satisfy the SBVR constraints.
+INSERT INTO "cpu architecture" ("slug") VALUES ('unknown');
```

Comment:
@Page- if sbvr is ran first the table should exist, but I see some migrations where we create a table as part of it. Is that necessary here? I just want to add a dummy cpu architecture if there is none, so we can satisfy the foreign key requirement. If you have a better suggestion on how to handle such case, I'm all ears :)

Reply:
We create the table if we need to insert data/modify it as part of the migration, in this case it should not be necessary as long as the various "keyframes" in the open-balena-api chain are deployed in order. Having the create table explicitly won't cause any issues though as long as you include the `IF NOT EXISTS`.

Also if you want this cpu arch to always be populated you should consider adding it to `balena-init.sql` as well